shared/components/splash-example.jsx

```diff
@@ -25,10 +32,8 @@ export default React.createClass({
     },
 
     render: function () {
-        var {FormattedMessage} = ReactIntl;
-
         var currentLocale = this.state.currentLocale;
-        var photosMessage = this.props.messages[currentLocale].photos;
+        var photosMessage = this.props.messages[currentLocale].photosNested;
```

Comment:
Currently only `en-US` has a translation for this variation of the message, we need to add it for all the locales (even if we don't use this PR.)